contracts/lst-staking/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+#![no_std]
+
+multiversx_sc::imports!();
+
+mod esdt;
+use esdt::{AvailableAmount, Esdt};
+
+#[multiversx_sc::contract]
+pub trait LstStakingContract {
+    #[init]
+    fn init(&self, unbond_period: u64) {
+        self.unbond_period().set(unbond_period);
```

Comment:
Might be good to have an `only_owner` endpoint to set this storage. Never know when you need to change it and will require a new contract upgrade